### PR TITLE
gh-68966: Document mailcap shell injection vulnerability

### PR DESCRIPTION
**Pull request by vstinner**

### CODE REVIEW

File: Doc/library/mailcap.rst [hunk 1]
```diff
@@ -26,6 +26,16 @@ The mailcap format is documented in :rfc:`1524`, "A User Agent Configuration
 Mechanism For Multimedia Mail Format Information", but is not an internet
 standard.  However, mailcap files are supported on most Unix systems.
 
+.. _ailcap-shell-injection:
+
+.. warning::
+
+   The Python implementation is vulnerable to shell command injection. By
+   design, the mailcap format uses shell commands. The caller is responsible to
+   validate the filename. The caller can create temporary filename, using
```
Comment (encukou) [inline Doc/library/mailcap.rst]: It's not only about the filename. The MIME type and parameter list need to be validated as well.
Also, how should they be validated? That's not clear.

Comment (vstinner) [reply Doc/library/mailcap.rst]: You're right, I rephrased the warning.

Comment (vstinner) [reply Doc/library/mailcap.rst]: "The caller is responsible to validate and sanitize input arguments."

File: Doc/library/mailcap.rst [hunk 2]
```diff
@@ -26,6 +26,18 @@ The mailcap format is documented in :rfc:`1524`, "A User Agent Configuration
...
 
+.. _mailcap-shell-injection:
+
+.. warning::
+
+   The Python implementation is vulnerable to shell command injection. By
+   design, the mailcap format uses shell commands. The caller is responsible to
+   validate and sanitize input arguments.
+
+   For the filename, the caller can create temporary filename, using
```
Comment (JelleZijlstra) [inline Doc/library/mailcap.rst]:
```suggestion
   For the filename, the caller can create a temporary filename, using
```

**Comment by vstinner** [comment]
https://github.com/python/cpython/pull/91993 got merged, I close this issue.

**Pull request closed without merging**